changed.d

```diff
@@ -444,12 +445,15 @@ Please supply a bugzilla version
             revRange.getBugzillaChanges.writeBugzillaChanges(w);
     }
 
+    w.put("$(CHANGELOG_FOOTER)\n");
     w.formattedWrite("$(CHANGELOG_NAV_LAST %s)\n", previousVersion);
 
     // write own macros
     w.formattedWrite(`Macros:
     VER=%s
     TITLE=Change Log: $(VER)
+    CHANGELOG_HEADER=
+    CHANGELOG_FOOTER=
```

Comment:
Won't this override anything specified elsewhere?

Reply:
Files which are seen later by Ddoc take a higher precedence and in this case, this is actually needed to avoid the unknown macro warning. I just though I save myself the hop via dlang.org this time, but as I [just realized](https://github.com/dlang/dlang.org/pull/1977#issuecomment-351555857) it's better if the contributors are directly emitted from the `changed.d` script.